### PR TITLE
add containers troubleshooting

### PR DESCRIPTION
**Pull request by cswatt**
new version of https://github.com/DataDog/documentation/pull/17345

### CODE REVIEW

File: content/en/containers/troubleshooting/_index.md [hunk 1]
```diff
@@ -2,8 +2,171 @@
...
+- The metrics endpoint is exposed and is open for the the Agent to reach.
+
+- There are no proxies or firewalls that might impede the Agent from accessing the endpoint. 
+
+- Agent has [Autodiscovery][10] enabled.
+
+
+### Logs are not collected
+
+There are two [environment variables][8] that can impact if logs will be collected and from which containers:
```
Comment (rtrieu) [inline content/en/containers/troubleshooting/_index.md:44]: did you mean the environment variables can...

have an impact? be impacted? impact something?

**Review by rtrieu — CHANGES_REQUESTED** [review]
Minor feedback, but otherwise looks good!

File: content/en/containers/troubleshooting/_index.md (hunk 1 above)
Comment (cswatt) [reply content/en/containers/troubleshooting/_index.md:44]: good call gonna replace it with "affect"

**Review by rtrieu — APPROVED** [review]
Looks good!

**Pull request merged**